opba-protocols/sandboxes/hbci-sandbox/src/main/java/de/adorsys/opba/protocol/sandbox/hbci/protocol/RequestStatusUtil.java

```diff
@@ -14,4 +14,8 @@ public boolean isForTransactionListing(Map<String, String> requestData) {
     public boolean isForAccountListing(Map<String, String> requestData) {
         return "HKSPA".equals(MapRegexUtil.getDataRegex(requestData, "TAN2Step\\d*\\.ordersegcode"));
     }
+
+    public boolean isForPaymentListing(Map<String, String> requestData) {
```

Comment:
This function is rather `isForPayment` and not `isForPaymentListing` - it executes payment